app/questionnaire/router.py

```diff
@@ -257,7 +278,7 @@ def get_enabled_section_keys(self):
                 section_key = (section_id, None)
                 yield section_key
 
-    def _get_first_incomplete_section_key(self):
+    def _get_first_incomplete_section_key(self) -> Sequence:
         for section_id, list_item_id in self.get_enabled_section_keys():
             if not self._progress_store.is_section_complete(section_id, list_item_id):
                 return section_id, list_item_id
```

Comment:
This returns an optional tuple of strings.

Reply:
Done, thanks!